master/buildbot/test/unit/test_reporter_http.py

```diff
@@ -77,7 +77,7 @@ def test_basic(self):
         yield self.createReporter()
         build = yield self.setupBuildResults(SUCCESS)
         build['complete'] = False
-        self.sp.buildStarted(("build", 20, "started"), build)
+        self.sp.buildStarted(("build", 20, "new"), build)
```

Comment:
I am a little bit reluctant to take coding style rule that is not enforced by an automated tool (or better fixed atomatically by autopep8)